architecture/core_concepts/routes.adoc

```diff
@@ -744,7 +744,7 @@ spec:
 termination.
 <3> The `*destinationCACertificate*` field specifies a CA certificate to
 validate the endpoint certificate, securing the connection from the router to
-the destination. This field is required, but only for re-encryption.
+the destination. This field is required for re-encryption. However, it can be left empty to use a configurable default destination CA certificate path. For example, an existing router, logging, metrics, web console, or registry console certificate.
```

Comment:
The "for example" probably doesn't make sense in this case.

Starting in 3.6, the router now automatically leverages the CA that is generated for service serving certificates (and is injected into every pod as `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt`) for routes that don't have a destinationCACertificate set.  This allows routes to be created that leverage end-to-end encryption without generating a certificate up front for the route.  Clusters where custom routers or the F5 router are in use may not support destinationCACertificates and will not admit the route unless the administrator has allowed that.